File: hello-node-lambda/handler.js
require("dotenv").config();

const result = require("./result");

module.exports.helloWorld = (event, context, callback) => {
    console.log(process.env.MY_VAR);

    console.log(event.queryStringParameters);

    const response = {
        statusCode: 200,
        headers: {
            "Access-Control-Allow-Origin": "*", // Required for CORS support to work
        },
        body: JSON.stringify({ ...result, ...event.queryStringParameters }),
    };

    callback(null, response);
};

File: hello-python-api-lambda/handler.py
import json


def hello(event, context):
    response = {
        "statusCode": 200,
        "body": json.dumps(event['queryStringParameters'], indent=4, sort_keys=True, ensure_ascii=False)
        # "body": json.dumps(body, indent=4, sort_keys=True, ensure_ascii=False)
    }

    if event['queryStringParameters'] == None:
        print('No parameters')
        
        response = {
            "statusCode": 403,
            "body": "Not allowed"
        }

    return response

